app/src/main/java/org/simple/clinic/scanid/ScanSimpleIdUpdate.kt

```diff
@@ -40,19 +41,26 @@ class ScanSimpleIdUpdate @Inject constructor(
       is ScannedQRCodeJsonParsed -> scannedQRCodeParsed(model, event)
       InvalidQrCode -> next(model.notSearching().invalidQrCode())
       is OnlinePatientLookupWithIdentifierCompleted -> onlinePatientLookupWithIdentifierCompleted(model, event)
+      is CompleteMedicalRecordsSaved -> patientsFoundByOnlineLookup(event.completeMedicalRecords)
     }
   }
 
   private fun onlinePatientLookupWithIdentifierCompleted(
       model: ScanSimpleIdModel,
       event: OnlinePatientLookupWithIdentifierCompleted
   ): Next<ScanSimpleIdModel, ScanSimpleIdEffect> {
-    return when(event.result) {
-      is NotFound, is OtherError -> next(model.notSearching(), OpenPatientSearch(event.identifier, null, model.patientPrefillInfo))
-      else -> noChange() // update the list of patients or patient received here
+    return when (event.result) {
+      is NotFound -> next(model.notSearching(), OpenPatientSearch(event.identifier, null, model.patientPrefillInfo))
```

Comment:
Afaig aren't we still doing the same thing in `NotFound` and `OtherError`? Why did we separate these out?

Reply:
Yes I had. I think while merging this went back. Will do this.